api/desecapi/tests/test_rrsets_bulk.py

```diff
@@ -175,6 +176,21 @@ def test_bulk_patch_does_not_need_ttl(self):
             status.HTTP_200_OK,
         )
 
+    def test_bulk_patch_delete_non_existing_rr_sets(self):
+        self.assertResponse(
+            self.client.bulk_patch_rr_sets(
+                domain_name=self.my_empty_domain.name,
+                payload=[
+                    {'subname': 'a', 'type': 'A', 'records': [], 'ttl': 22},
+                    {'subname': 'b', 'type': 'AAAA', 'records': []},
+                ]),
+            status.HTTP_400_BAD_REQUEST,
+            [
+                {'records': ['On creation, must be a non-empty list.']},
```

Comment:
uh. Do we now consider this a creation case? Previously, the "deletion" characteristics had precedence.

I still think that `PATCH` should be idempotent, i.e. its payload describes the target state, and if you repeat the request, the state after the repetition is the same as before the repetition, with a `2xx` status. If you disagree, let's discuss it a bit more to see what's better.

Reply:
Let's make all delete operation idempotent